brokerapi/brokers/cloudsql/broker.go

```diff
@@ -274,6 +205,92 @@ func (b *CloudSQLBroker) Provision(instanceId string, details models.ProvisionDe
 
 }
 
+func createFirstGenRequest(planDetails, params map[string]string) googlecloudsql.DatabaseInstance {
+	// set up instance resource
+	return googlecloudsql.DatabaseInstance{
+		Settings: &googlecloudsql.Settings{
+			IpConfiguration: &googlecloudsql.IpConfiguration{
+				RequireSsl:  true,
+				Ipv4Enabled: true,
+			},
+			Tier:             planDetails["tier"],
+			PricingPlan:      planDetails["pricing_plan"],
+			ActivationPolicy: params["activation_policy"],
+			ReplicationType:  params["replication_type"],
+		},
+		DatabaseVersion: params["version"],
+		Region:          params["region"],
+	}
+}
+
+func createSecondGenOrPostgresRequest(planDetails, params map[string]string) (googlecloudsql.DatabaseInstance, error) {
```

Comment:
this name is a lot. can it just be: createRequest and think of the first gen as a special case? this method doesn't contain anything postgres specific.

Reply:
ack